changelog/fragments/1751485154-Add-missing-multiplatform-builds.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: Add missing multiplatform builds
```

Comment:
Does this need a changelog? Is this user facing? Have we released this on a final build?

Reply:
good point; this only got to 9.1 and 8.19, i'll remove the changelog